nifi-extension-bundles/nifi-aws-bundle/nifi-aws-kinesis/src/main/resources/docs/org.apache.nifi.processors.aws.kinesis.ConsumeKinesis/additionalDetails.md

```diff
@@ -0,0 +1,35 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+# ConsumeKinesis
+
+## Record processing
+
+When _Record Reader_ and _Record Writer_ properties are set, _ConsumeKinesis_ operates in Record mode.
+In this mode, the processor reads records from Kinesis streams using the configured _Record Reader_,
+and writes them to FlowFiles using the configured _Record Writer_.
```

Comment:
This should be removed and an explicit property added, following the `Processing Strategy` property approach in `ConsumeKafka`.